### PR TITLE
getting rid of jupyter notebooks 🎉

### LINKED ISSUE #151: Use of jupytext for version control
[jupytext](https://github.com/mwouts/jupytext) allows to keep simple text file formats that are readed by notebooks if with the addon. It offers multiple options, one native to the plugin and others like md, shinx-gallery, etc. It doesn't save the output of the cells though!

### PR DESCRIPTION
**Pull request by dpshelio**
Finally, we've got rid of jupyter notebooks. Thanks to jupytext we don't need to change much of the infrastructure as it allows to open the `py:percent` format as jupyter notebooks and keep the original python file in sync.  I've got it build under my fork: https://dpshelio.github.io/rsd-engineeringcourse/ and I haven't noticed any problem with it.

This closes #151

Additionally, I've fixed the jekyll-theme so that it also show the menu of the sections when browsed from a mobile device.
https://github.com/dpshelio/indigo-jekyll/commit/46c1774f11004ae068bb3328002967d944391538

(to be merged after #228)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[tool.jupytext]
+formats = "ipynb,py:percent"
+notebook_metadata_filter="kernelspec,jupytext,jekyll"
```
Comment (p-j-smith) [inline pyproject.toml:3]:
```suggestion
notebook_metadata_filter="jupytext,jekyll"
```

**Review by p-j-smith — CHANGES_REQUESTED** [review]
very cool!! It all seems to work okay - I tried it out locally, editing a notebook and the .py files were updated correctly 🎉 

Just one small suggestion to remove the `kernelspec` from the notebook metadata - if contributors have a different kernel name from you then that would show up as a change in any pr

File: pyproject.toml (hunk 1 above)
Comment (dpshelio) [reply pyproject.toml:3]: I tried that last week and again, and it didn't work. It keeps saving the `kernelspec`. Would you mind giving it a try and see whether it works for you? There may be a flag somewhere that I'm missing.

Comment (dpshelio) [reply pyproject.toml]: the trick was using `-kernelspec`

Comment (p-j-smith) [reply pyproject.toml]: Oh nice!

**Review by p-j-smith — APPROVED** [review]

**Pull request merged**